distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -175,28 +183,86 @@ public void testCswRegistryCreateDup() throws Exception {
                 .post(CSW_PATH.getUrl());
     }
 
-    @Ignore
     @Test
     public void testCswRegistryStoreCreate() throws Exception {
-        //Stub test, waiting for additional features/service before implementing
-        //Use federation admin to create a registry entry
-        //on a 'remote' (aka loopback) catalog store
+        org.codice.ddf.security.common.Security.runAsAdmin(() -> {
```

Comment:
After talking with Chris the reasoning behind using `FederationAdminServiceImpl ` is used is to test the functionality of the RegistryStore. Currently, there is not a REST endpoint that provides CUD operations with a destination which is required to test the RegistryStore, making the RegistryStore only reachable through  `FederationAdminServiceImpl`